src/convertToObject.js

```diff
@@ -6,7 +6,38 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // make an array with strings of sourceString
+  const properties = [];
+
+  let temp = sourceString.trim() + ';';
+
+  while (temp.includes(';')) {
+    properties.push(temp.slice(0, temp.indexOf(';') + 1));
+    temp = temp.slice(temp.indexOf(';') + 1).trim();
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. It will decrease code readability. The variable `properties` could be more descriptive, indicating that it holds CSS declarations.